test/e2e/network/util_iperf.go

```diff
@@ -97,6 +97,8 @@ func (s StrSlice) get(i int) string {
 
 // intOrFail is a convenience function for parsing integers.
 func intOrFail(debugName string, rawValue string) int64 {
+	// Trim newline in raw string before parsing it
+	rawValue = strings.Replace(rawValue, "\n", "", -1)
```

Comment:
This doesn't seem great to me. We should be Trim()ing the string before handing to this function. Also, this function makes 

"1\n2\n3\n5" become a valid integer.

Reply:
Fixed. PTAL. Thanks!